dice/apps/rounds/tests/test_views.py

```diff
@@ -55,12 +56,12 @@ def test_roll(self):
             ],
             format='json',
         )
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     def test_too_many_roll(self):
-        """Ensure http 403 is returned after third roll."""
+        """Ensure http 409 is returned after third roll."""
 
-        for status_code in [200, 200, 403]:
+        for status_code in [status.HTTP_200_OK, status.HTTP_200_OK, status.HTTP_409_CONFLICT]:
```

Comment:
NIP: let's read about [`subTest`](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests) because it will be nice to use it here